lib/showoff.rb

```diff
@@ -303,25 +304,25 @@ def process_markdown(name, section, content, opts={:static=>false, :pdf=>false,
       # todo: unit test
       lines = content.split("\n")
       @logger.debug "#{name}: #{lines.length} lines"
-      slides = []
-      slides << (slide = Slide.new)
+      slideset = []
```

Comment:
Also to answer your question. I didn't do a s/slide/slideset/ I actually manually looked through. I only changed it in places like this where clearly "slides" is getting assigned to something.